src/adapters/poi_popup.js

```diff
@@ -129,7 +129,9 @@ PoiPopup.prototype.setPopupPosition = function(event, popupOptions) {
 };
 
 PoiPopup.prototype.close = function() {
-  this.popupHandle.remove();
+  if (this.popupHandle) {
+    this.popupHandle.remove();
```

Comment:
Shouldn't you change `this.popupHandle` value to avoid it being still "available"?

Reply:
Badly understood the condition...